lib/active_merchant/billing/gateways/d_local.rb

```diff
@@ -228,6 +252,24 @@ def message_from(action, response)
         response['status_detail'] || response['message']
       end
 
+      def success_from_get(response)
+        if response.instance_of? Array || (response['status_code'].nil? && response['error_code'].nil?)
+          return true
+        else
+          return false
```

Comment:
Will checking the status_code of the response be enough for checking if the API was successful?

Reply:
Yes I feel harish because Earlier code says the same.